components/ParliamentaryGroupCard/index.js

```diff
@@ -0,0 +1,39 @@
+import * as CUI from '@chakra-ui/react';
+
+const ParliamentaryGroupCard = ({
+  logoParty = 'https://cdn.mercadonegro.pe/wp-content/uploads/2020/01/congreso-2020-alianza-para-el-progreso-1024x1024.png',
+  nameParty = 'Alianza por el Progreso',
+  members = '12',
+}) => {
+  return (
+    <CUI.Flex
+      direction="column"
+      maxW={44}
+      minH="244px"
+      px={18.5}
+      pt="20px"
+      pb={6}
+      textAlign="center"
+      boxShadow="sm"
+      border="1px solid"
+      borderColor="secondary.200"
+      rounded="4">
+      <CUI.Image w={16} mx="auto" src={logoParty} />
```

Comment:
```suggestion
      <CUI.Image w="16" mx="auto" src={logoParty} />
```
Así para todos los números porfa.